unix/vncserver.in

```diff
@@ -682,6 +682,14 @@ sub CheckDisplayNumber
     }
     close(S);
 
+    if (open(SOCKS, "</proc/net/unix")) {
```

Comment:
The file handle SOCKS is new here. We can reuse S like the previous checks without problems because it is closed in line 683. I like that the test is skipped if the open fails. Everything else looks fine.